codestyle/styles.md

```diff
@@ -0,0 +1,672 @@
+# CSS/LESS code style
```

Comment:
I thought about it, but decided to don't write this section since it is beyond the scope of the document. The point you mentioned is more related to the project files structure than to coding conventions.